autofaiss/indices/distributed.py

```diff
@@ -355,7 +378,315 @@ def add_embeddings_to_index(
                 src_folder=next_stage_folder,
                 dst_folder=final_folder,
                 index_optimizer=index_optimizer,
+                make_direct_map=make_direct_map
             )
             if fs.exists(temporary_indices_folder):
                 fs.rm(temporary_indices_folder, recursive=True)
             return None, metrics
+
+
+def _add_embeddings_to_index_from_hdfs(
```

Comment:
Same remark for this function if it can work for other remote file system.

Reply:
Done